modules/data.atmosphere/R/download.NARR_site.R

```diff
@@ -322,25 +322,25 @@ generate_narr_url <- function(dates, flx) {
   )
   tibble::tibble(date = dates) %>%
     dplyr::mutate(
-      year = lubridate::year(date),
-      month = lubridate::month(date),
-      daygroup = daygroup(date, flx)
+      year = lubridate::year(.data$date),
+      month = lubridate::month(.data$date),
+      daygroup = daygroup(.data$date, .data$flx)
```

Comment:
`flx` here is an argument to the parent function, not a variable in the data frame. I missed this on my first review.

```suggestion
      daygroup = daygroup(.data$date, flx)
```

Reply:
@ashiklom fixed this.